.travis.yml

```diff
@@ -154,7 +154,7 @@ jobs:
         # Set MAVEN_OPTS for Surefire launcher. Skip remoteresources to avoid intermittent connection timeouts when
         # resolving the SIGAR dependency.
         - >
-          MAVEN_OPTS='-Xmx800m' ${MVN} test -pl ${MAVEN_PROJECTS}
+          MAVEN_OPTS='-Xmx1100m' ${MVN} test -pl ${MAVEN_PROJECTS}
```

Comment:
what is this change for?

Reply:
Current UTs don't cover load segment in a lazy way. This PR add a UT to test lazy loading which needs extra memory.

Reply:
:+1: was just surprised how much more it needed, if travis passes then I assume the change is fine, was just curious mostly